### PR TITLE
Update readme to include project scope.

### PR DESCRIPTION
**Pull request by chexxor**
I wanted to include this project's scope in the README. LMK what you think.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,30 @@
...
-Welcome to the PureScript documentation repository! This repository is meant to be a collaborative effort, so please feel free to add/edit content where it is appropriate to do so.
+Welcome to the PureScript documentation repository!
 
-## Introduction
-
-PureScript is a small strongly typed programming language that compiles to JavaScript.
+PureScript is a small, strongly typed programming language that compiles to JavaScript.
 To get a better overview of PureScript, visit [The PureScript Website](http://purescript.org).
 
+This repository is meant to be a collaborative effort, so please feel free to make a pull request to add/edit content or create an issue to discuss it. Without you, PureScript would have *no* documentation! Thanks for helping!
```
Comment (LiamGoodacre) [inline README.md]: `Without you, PureScript would have *no* documentation!` this phrasing sounds a little like a dig towards feature contributors not documenting their work, I doubt you meant it that way though.

**Comment by LiamGoodacre** [comment]
On the whole, it sounds great!

File: README.md (hunk 1 above)
Comment (hdgarrood) [reply README.md]: Yeah I have to say I'm not hugely fond of this sentence either. To me it reads as saying "we [contributors] haven't done the work of documenting PureScript, we're leaving it to everyone else", which... isn't really true - the evidence being the existence of this repo!

Comment (hdgarrood) [reply README.md]: But other than that it looks good to me too.

Comment (chexxor) [reply README.md]: @LiamGoodacre Well, I, myself, don't expect feature contributors to document their work. Adding features is hard enough! Expecting them to also come over here and add something seems a bit much to ask. I'm not much of a contributor to the PS compiler, myself, so I might be too lenient.

I'll update it - I see a better way to say what I meant.

**Comment by chexxor** [comment]
I fixed that phrasing, so a follow-up review is due.

**Pull request merged**